plugins/modules/nd_setup.py

```diff
@@ -248,6 +255,31 @@
                 - The ASN of the BGP peer.
                 type: int
                 required: true
+  deployment_mode:
+    description:
+    - This is used for enabling the available services between
+      Orchestrator, Fabric Controller and Insights during the intial installation.
+    - This option is only applicable for ND version 3.1.1 and later.
+    type: list
+    elements: str
+    choices: [ ndo, ndfc, ndi ]
+  external_services:
+    description:
+    - The persistent Service IPs/Pools to provide
+      if I(deployment_mode) includes C(ndfc) or C(ndi).
```

Comment:
Is "if I(deployment_mode) includes C(ndfc) or C(ndi)." required here?

Reply:
It is not required but could be phrased differently. instead of "If..." maybe something like "This is used when I(deployment_mode) includes C(ndfc) or C(ndi)."

Reply:
I think this sounds good:
"This can be used when I(deployment_mode) is C(ndfc) or C(ndi)"